client/components/button/test/index.js

```diff
@@ -0,0 +1,1064 @@
+/**
+ * Created by nitrajka on 09/10/16.
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../';
+import Gridicon from 'components/gridicon';
+
+describe( 'Button', function() {
+	describe( 'not compact', function() {
+		describe( 'button tag', function() {
+			const text = 'Text';
+			const icon = ( <Gridicon size={ 18 } icon="arrow-left" /> );
+			const disabled = 'Disabled';
+
+			describe( 'basic styled', function() {
+				it( 'renders with text', function() {
+					const button = shallow( <Button>{ text }</Button> );
+					expect( button ).to.contain( 'Text' );
```

Comment:
Again, I'm not sure that assigning the string to a variable adds much value, but if you were to keep it, I'd make sure that the `contain` assertion here checks against the variable, not the text itself (e.g. what if the variable were to change? we'd expect all tests to still pass).